services/s3/src/main/resources/software/amazon/awssdk/services/s3/execution.interceptors

```diff
@@ -12,4 +11,4 @@ software.amazon.awssdk.services.s3.internal.handlers.EnableTrailingChecksumInter
 software.amazon.awssdk.services.s3.internal.handlers.ExceptionTranslationInterceptor
 software.amazon.awssdk.services.s3.internal.handlers.GetObjectInterceptor
 software.amazon.awssdk.services.s3.internal.handlers.CopySourceInterceptor
-software.amazon.awssdk.services.s3.internal.handlers.SignerOverrideInterceptor
```

Comment:
Is the SignerOverrideInterceptor obsoleted by the endpoint auth scheme? 
Should it be removed from the code base too?

Reply:
Yes